docs/_docs/01_getting-started/supported-versions.md

```diff
@@ -27,6 +35,9 @@ The officially supported versions are:
 | 0.12.x            | [0.19.0](https://github.com/gruntwork-io/terragrunt/releases/tag/v0.19.0) - [0.24.4](https://github.com/gruntwork-io/terragrunt/releases/tag/v0.24.4) |
 | 0.11.x            | [0.14.0](https://github.com/gruntwork-io/terragrunt/releases/tag/v0.14.0) - [0.18.7](https://github.com/gruntwork-io/terragrunt/releases/tag/v0.18.7) |
 
+Due to the HashiCorp BSL license change, we are restricting Terraform to the latest open source version (`1.5.7`).
+Going forward we will be adding support for OpenTofu. To learn more, see the official [OpenTofu website](https://opentofu.org/)
+and [project status](https://github.com/opentofu/opentofu/blob/main/WEEKLY_UPDATES.md).
```

Comment:
Terragrunt should keep working with newer Terraform versions, even BSL ones. It's an open source wrapper, and shouldn't be an issue with BSL.

Reply:
Will update implementation to work with BSL Terraform